### PR TITLE
Create `npm run lint` for easier linting

### PR DESCRIPTION
**Pull request by dbolson**
Based on comments in #3, I thought this could make linting easier.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,10 +28,11 @@
     "react-scripts": "1.0.7"
   },
   "scripts": {
-    "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "flow": "flow"
+    "flow": "flow",
+    "lint": "eslint src/",
```
Comment (dbolson) [inline package.json]: Added this, alphabetized the rest.

File: src/components/header/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { connect } from 'react-redux'
 import _ from 'lodash'
 
-import { withRouter} from 'react-router-dom'
+import { withRouter } from 'react-router-dom'
```
Comment (dbolson) [inline src/components/header/index.js]: Fixed a linting error!

File: README.md [hunk 3]
```diff
@@ -19,6 +19,7 @@
 * [Async action](http://redux.js.org/docs/advanced/AsyncActions.html) are handled with [`redux-thunk`](https://stackoverflow.com/questions/35411423/how-to-dispatch-a-redux-action-with-a-timeout/35415559#35415559). Keep side effects contained in action cretors.
 * Use [flow annotations](https://flow.org/en/docs/frameworks/react/) for all components.
 * [`recompose`](https://github.com/acdlite/recompose/blob/master/docs/API.md) contains many useful helpers.
+* Keep a consistent style by frequently running `npm run lint`.
```
Comment (jeffmay) [inline README.md]: I assume after running this, you should see nothing and there are no errors? If so, can you add that to the comment?

Comment (dbolson) [reply README.md]: Updated.

**Pull request merged**